build.gradle

```diff
@@ -340,6 +374,56 @@ dependencies {
 
   smokeTestImplementation sourceSets.main.runtimeClasspath
 
+  //Contract Testing Implementation - Start
+  contractTestImplementation group: 'org.springframework.boot', name: 'spring-boot-starter', {
+    exclude group: 'org.hamcrest', module: 'hamcrest-core'
+  }
+
+  contractTestImplementation group: 'org.springframework.boot', name: 'spring-boot-starter', {
+    exclude group: 'org.hamcrest', module: 'hamcrest-core'
+  }
+
+  contractTestImplementation group: 'net.serenity-bdd', name: 'serenity-rest-assured', version: versions.serenity
+  contractTestImplementation group: 'net.serenity-bdd', name: 'serenity-core', version: versions.serenity
+  contractTestImplementation group: 'net.serenity-bdd', name: 'serenity-junit', version: versions.serenity
+  contractTestImplementation group: 'net.serenity-bdd', name: 'serenity-spring', version: versions.serenity
+  contractTestImplementation group: 'au.com.dius', name: 'pact-jvm-consumer-junit5_2.12', version: versions.pact_version
+  contractTestImplementation group: 'au.com.dius', name: 'pact-jvm-consumer-java8_2.12', version: versions.pact_version
+  contractTestCompile group: 'au.com.dius', name: 'pact-jvm-provider-junit5', version: '4.0.10'
+  contractTestCompile group: 'au.com.dius', name: 'pact-jvm-provider-gradle', version: '4.0.10'
+
+  contractTestImplementation "org.junit.jupiter:junit-jupiter-api:5.6.2"
+  contractTestImplementation "org.junit.vintage:junit-vintage-engine:5.6.2"
+  contractTestCompile("org.junit.jupiter:junit-jupiter-api:5.6.2")
+  contractTestRuntime("org.junit.jupiter:junit-jupiter-engine:5.6.2")
+  contractTestImplementation group: 'org.junit.platform', name: 'junit-platform-launcher', version: '1.2.0'
+
+  contractTestCompile group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: '2.1.3.RELEASE'
+  contractTestImplementation group: 'io.rest-assured', name: 'xml-path', version: '4.2.0'
+  contractTestImplementation group: 'io.rest-assured', name: 'json-path', version: '4.2.0'
+
+  contractTestAnnotationProcessor 'org.projectlombok:lombok:1.18.12'
+  contractTestCompileOnly 'org.projectlombok:lombok:1.18.12'
+  contractTestImplementation "ch.qos.logback:logback-classic:1.2.3"
+
+  contractTestImplementation sourceSets.main.runtimeClasspath
+  contractTestImplementation sourceSets.test.runtimeClasspath
+
+  integrationTestImplementation ("org.springframework.boot:spring-boot-starter-test")
```

Comment:
As part of a ContractTestImplementation make sure that IntegrationTestImplementations are not referenced.